src/checkers/inference/model/ConstraintManager.java

```diff
@@ -158,6 +153,40 @@ public ExistentialConstraint createExistentialConstraint(Slot slot,
                 ifExistsConstraints, ifNotExistsConstraints, getCurrentLocation());
     }
 
+    private void commonArithmeticConstraintInputCheck(String operation, Slot lhs, Slot rhs,
```

Comment:
Could we replace the `String` type to an `Enum` type for `operation`? Maybe this `Enum` type can be an inner class for `ArithmeticConstraint`.

Reply:
Done